### ISSUE TITLE
:pathom/as doesn't work with batched resolvers

### ISSUE THREAD

**Issue by rschmukler** [wilkerlucio/pathom#89]
It seems that using `:pathom/as` in conjunction with `:pc/batch? true` results in only the first item getting the alias applied, while all others are resolved with the original key.

**Comment by rschmukler** [comment]
Just adding a minimum test case:

```clj
(pc/defresolver foo-resolver [env input]
  {::pc/input  #{}
   ::pc/output [{:foos [:foo-id]}]}
  {:foos (map #(hash-map :foo-id %) (range 10))})

(pc/defresolver param-resolver [env input]
  {::pc/input  #{:foo-id}
   ::pc/output [:n]
   ::pc/batch? true}
  (let [n (-> env p/params :n)]
    (mapv (constantly {:n n}) input)))

(def registry [foo-resolver param-resolver])

(def parser
  (p/parallel-parser
   {::p/env     {::p/reader               [p/map-reader
                                           pc/parallel-reader
                                           pc/open-ident-reader
                                           p/env-placeholder-reader]
                 ::p/placeholder-prefixes #{">"}
                 :log                     log
                 :db                      db}
    ::p/mutate  pc/mutate-async
    ::p/plugins [(pc/connect-plugin {::pc/register registry})
                 p/error-handler-plugin
                 p/request-cache-plugin
                 p/trace-plugin]}))

(<!! (parser {} `[{:foos [(:n {:pathom/as :a :n 5})]}])))
```

Yielding:

```clj
{:foos [{:a 5} {:n 5} {:n 5} {:n 5} {:n 5} {:n 5} {:n 5} {:n 5} {:n 5} {:n 5}]}
```

**Comment by wilkerlucio** [comment]
Fixed in 70d228195dba5fa981d1eb4fb4c6ab4eb44a952c
